index.html

```diff
@@ -2738,6 +2737,28 @@ <h4>Extensions</h4>
 				</section>
 			</section>
 		</section>
+		<section id="publication-resources">
+			<h2>Publication Resources</h2>
+
+			<p>The list of resources belonging to a <a>digital publication</a> &#8212; its <a>bounds</a> &#8212; is
+				obtained from the union of resources listed in the <a href="#default-reading-order"
+						><code>readingOrder</code></a> and <a href="#resource-list"><code>resources</code></a>,
+				including any <a href="#linkedresource-alternate"><code>alternate</code></a> resources. The process for
+				obtaining this list is described in <a href="#processing-pub-resources"></a>.</p>
+
+			<p>All other resources are outside the bounds of the digital publication (e.g., resources listed in the
+						<a><code>links</code></a> section and hyperlinks in the content to external resources on the
+				Web).</p>
+
+			<p>The only restriction this specification places on publication resources is that they be obtainable by a
+				URL that uses either the <code>http</code> or <code>https</code> scheme (see <a href="#value-url"></a>).
+					<a>Profiles</a> of this specification, however, MAY restrict both the content type and location of
+				resources.</p>
```

Comment:
What about relative URLs?

I'm also concerned about restricting this to `http` and `https`. Imagine if this spec were older and restricted it to just `http` URLs...and then `https` came along...i.e. not sure restricting scheme--when we don't also restrict a protocol or intended API usage--makes much sense.

Reply:
Can you open a new issue for this? It's a fair question, but it's not new to this PR, so not really the place to take it up.

Reply:
@BigBlueHat the process in the section is supposed to run _after_ the processing algorithm has run (it relies on the 'internal representation'). By that point the relative url-s expressed in the manifest have been turned into absolute ones. Ie, that part of the comment should be o.k.

However, I fully agree with the second comment. It actually may also create issues with off-line UA implementations which may, in URI terms, rely on `file:` URIs, in fact, because everything is within a package. So yes, it is clearly an issue to discuss and I agree with @mattgarrish that this should be the subject of a separate issue, and is orthogonal to this PR.

Reply:
> off-line UA implementations which may, in URI terms, rely on `file:` URIs

Right, I think the restriction to http(s) was heavily dependent on the "web" publications underpinning.